src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -1,271 +1,332 @@
-#!/usr/bin/env python
```

Comment:
For this file, you've changed the line endings.  Please run this through "dos2unix" or the equivalent and commit the change.  

Otherwise, this is  difficult to review.

Reply:
Please let me know if this is resolved in most recent push.